lib/connection_string.js

```diff
@@ -500,6 +493,46 @@ function checkTLSOptions(queryString) {
   }
 }
 
+/**
+ * Checks options object if both options are present (any value) will throw an error.
+ *
+ * @param {Object} options Options Object
```

Comment:
[_super_ nit]: prefer lowercase `object` as a type or `any` for our ts future 🚀

Reply:
:+1: